### ISSUE TITLE
Tablet (iPad) shows up as a desktop device in the people list

### ISSUE THREAD

**Issue by robertlong** [mozilla/hubs#3379]
**Description**
Tablet devices use the desktop device icon in the people list

**Comment by vincentfretin** [comment]
Yeah probably because iOS Safari now uses desktop view by default. If you switch to Mobile view, does the icon change?
I know the user agent string changes between those two views.
